test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/SimpleHttpValueProvider.cs

```diff
@@ -59,11 +59,28 @@ public Task<bool> ContainsPrefixAsync(string prefix)
 
         public Task<ValueProviderResult> GetValueAsync(string key)
         {
-            ValueProviderResult result = null;
+            ValueProviderResult result = ValueProviderResult.None;
+
```

Comment:
A bit unrelated, but we should probably rename this. There's nothing `Http` about it (anymore).